examples/verif-manual/vm-011-residual-stress-problem.py

```diff
@@ -0,0 +1,174 @@
+    :width: 400
+    :alt:  VM11 Problem Sketch
+    :figclass: align-center
+
+    **VM11 Problem model**
+
+Material Properties
+ - :math:`\sigma_{yp} = 30,000\,psi`
+ - :math:`E = 30 \cdot 10^6\,psi`
+
+.. figure:: ../_static/vm11_setup_2.png
+    :align: center
+    :width: 400
+    :alt:  VM11 Material Model
+    :figclass: align-center
+
+    **VM11 Material Model**
+
+Geometric Properties:
+ - :math:`A = 1\,in^2`
+ - :math:`l = 100\,in`
+ - :math:`\Theta = 30°`
+
+Loading:
+ - :math:`F_1 = 51,961.5\,lb`
+ - :math:`F_2 = 81,961.5\,lb`
+
+Analysis Assumptions and Modeling Notes:
+ - Automatic load stepping (:meth: Mapl.autots <ansys.mapdl.core.Mapdl.autots>,ON)
+   is used to obtain the nonlinear plastic solution (load steps 2 and 3).
+
+"""
+# sphinx_gallery_thumbnail_path = '_static/vm11_setup.png'
+
+import math
+
+from ansys.mapdl.core import launch_mapdl
+
+###############################################################################
+# Start MAPDL
+# ~~~~~~~~~~~
+# Start MAPDL.
+
+mapdl = launch_mapdl()
+mapdl.clear()  # optional as MAPDL just started
+
+###############################################################################
+# Pre-processing
+# ~~~~~~~~~~~~~~
+# Enter verification example mode and the pre-processing routine.
+
+mapdl.verify("vm11")
+mapdl.prep7()
+mapdl.title("VM11 RESIDUAL STRESS PROBLEM")
+
+###############################################################################
+# Define element type
+# ~~~~~~~~~~~~~~~~~~~
+# Set up the element type ``LINK180``.
+
+# Type of analysis: Static.
+mapdl.antype("STATIC")
+
+# Element type: LINK180.
+mapdl.et(1, "LINK180")
+mapdl.sectype(1, "LINK")
+mapdl.secdata(1)
+mapdl.mp("EX", 1, 30e6)
+mapdl.tb("PLAS", 1, "", "", "BKIN")  # TABLE FOR BILINEAR KINEMATIC HARDENING
+mapdl.tbtemp(100)
+mapdl.tbdata(1, 30000)  # YIELD STRESS
+
+# Print
+print(mapdl.mplist())
+
+###############################################################################
+# Define model geometry
+# ~~~~~~~~~~~~~~~~~~~~~
+# Set up parameters and geometry.
+
+L = 100
+theta = 30
+xloc = L * math.tan(math.radians(theta))
+mapdl.n(1, -xloc)
+mapdl.n(3, xloc)
+mapdl.fill()
+mapdl.n(4, "", -L, mute=True)
+
+###############################################################################
+# Define elements
+# ~~~~~~~~~~~~~~~
+# Create elements.
+mapdl.e(1, 4)
+mapdl.e(2, 4)
+mapdl.e(3, 4)
+mapdl.outpr("", 1)
+mapdl.d(1, "ALL", "", "", 3)
+mapdl.f(4, "FY", -51961.5)  # APPLY LOAD F1
+mapdl.finish()
+
+###############################################################################
+# Solve
+# ~~~~~
+# Enter solution mode and run the simulation.
+mapdl.slashsolu()
+mapdl.solve()
+mapdl.finish()
+
+###############################################################################
+# Post-processing
+# ~~~~~~~~~~~~~~~
+# Enter post-processing.
+
+# Enter the post-processing routine.
+mapdl.post1()
+
+q = mapdl.queries
+bot_node = q.node(0, -100, 0)
+def_node = mapdl.get("_", "NODE", bot_node, "U", "Y")
```

Comment:
```suggestion
bot_node = q.node(0, -100, 0)
def_node = mapdl.get_value("NODE", bot_node, "U", "Y")
```

Reply:
[The documentation](https://mapdl.docs.pyansys.com/version/stable/api/_autosummary/ansys.mapdl.core.Mapdl.get.html) say that `par` it is optional, but I get an error when doing this.
```
Unknown label for *GET command= 4
  The *GET command is ignored.
```

I also have looked in [pymapdl source code](https://github.com/ansys/pymapdl/blob/main/src/ansys/mapdl/core/mapdl.py#L2383) and I see nothing making this optional

I have found the function [mapdl.get_value](https://mapdl.docs.pyansys.com/version/stable/api/_autosummary/ansys.mapdl.core.Mapdl.get_value.html) in replacement.